### PR TITLE
week 18, project mongo api - Malin Göthe

### PR DESCRIPTION
**Pull request by M4L5**
The routes are not working. i will try to fix it but not sure if i'm gonna make it before deadline

https://goethe-mongodb-api.herokuapp.com/

### CODE REVIEW

**Review by lars-soderman — APPROVED** [review]
Malin! It was fun looking at your project, i think you have done a real good job. A few things there that i didnt fully understand and some commented out code that could be removed, but overall i give this a big thumbs up 👍  I also got some good inspiration for how i can improve my own project. 🎸

File: server.js [hunk 1]
```diff
@@ -1,38 +1,91 @@
...
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const MusicSchema = new mongoose.Schema({
+  "id": Number,
```
Comment (KarolCislowski) [inline server.js:14]: Why do you need both id and _id?

File: server.js [hunk 2]
```diff
@@ -1,38 +1,91 @@
...
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
+
+app.use((req, res, next) => {
+  if(mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: 'Service not available' })
+  }
```
Comment (KarolCislowski) [inline server.js:58]: Nice connection check 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**